examples/kubernetes/nfs-provisioner/README.md

```diff
@@ -8,11 +8,7 @@
 kubectl create -f https://raw.githubusercontent.com/kubernetes-csi/csi-driver-nfs/master/examples/kubernetes/nfs-provisioner/nfs-server.yaml
```

Comment:
One more nit, can we name this directory "nfs-server" instead so we don't confuse it with dynamic provisioning?

Reply:
fixed, PTAL